src/components/Gallery/__stories__/Gallery.stories.tsx

```diff
@@ -63,7 +66,7 @@ const ImagesGalleryTemplate: StoryFn<GalleryProps> = () => {
                 {images.map((image, index) => (
                     <GalleryItem
                         key={index}
-                        {...getGalleryItemImage({src: image.url, name: image.name})}
+                        {...getGalleryItemImage({src: image.url, name: image.name, mobile})}
```

Comment:
Why is it necessary to provide the isMobile argument to all the default renderers?  It would be better to create a separate component for the default item name and use useMobile hook inside